infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -306,21 +480,39 @@ sites:
     #   - varnish.main.kobenhavn.dplplat01.dpl.reload.dk
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHaTkDvjLW/b2qVj8FIvtX9x3TxFFZTENn+w2CFELeoC"
     <<: *default-release-image-source
+    primary-domain: "bibliotek.kk.dk"
+    secondary-domains:
+      - "www.bibliotek.kk.dk"
+    autogenerateRoutes: true
   koge:
     name: "KøgeBibliotekerne"
     description: "The library site for Køge"
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIMRB54Z7o6Vt3qOucjvE7No6LN2NdBNtfRi2f4y+UQdA"
     <<: *default-release-image-source
+    primary-domain: "koegebib.dk"
+    secondary-domains:
+      - "www.koegebib.dk"
+      - "køgebib.dk"
+      - "www.køgebib.dk"
```

Comment:
Vi har ikke testet om det her virker eller om det skal punycodes

Reply:
jeg prøvet teste det her nu

Reply:
Skal ændres til punycoded url for at virke